src/progress_bar.rs

```diff
@@ -489,6 +490,34 @@ impl ProgressBar {
     }
 }
 
+/// The alignment value for `MultiProgress`, the alignment controls how the multi progress
+/// is aligned if some its progress bars get removed.
+///
+/// E.g. `Top` alignment (default), when _progress bar 2_ is removed:
+/// ```ignore
+/// [0/100] progress bar 1        [0/100] progress bar 1
+/// [0/100] progress bar 2   =>   [0/100] progress bar 3
+/// [0/100] progress bar 3
+/// ```
+///
+/// `Bottom` alignment
+/// ```ignore
+/// [0/100] progress bar 1
+/// [0/100] progress bar 2   =>   [0/100] progress bar 1
+/// [0/100] progress bar 3        [0/100] progress bar 3
+/// ```
+#[derive(Debug, Copy, Clone)]
+pub enum MultiProgressAlignment {
+    Top,
+    Bottom,
+}
+
+impl Default for MultiProgressAlignment {
+    fn default() -> Self {
+        Self::Top
+    }
+}
+
```

Comment:
These definitions should probably not live in this module, let's move them into `draw_state`, below the `ProgressDrawState` definition.